pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -182,6 +182,12 @@ in
       fzf-lua
       plenary-nvim
     ];
+    nvimSkipModules = lib.optionals stdenv.hostPlatform.isDarwin [
+      # Tries to spawn a server on darwin and fails because "address already in use"
+      "advanced_git_search.fzf.previewers.init"
+      "advanced_git_search.fzf.pickers.init"
+      "advanced_git_search.fzf.pickers.utils"
+    ];
```

Comment:
Are you seeing the same issue as https://github.com/NixOS/nixpkgs/issues/431458

Would be a separate PR, too.

Reply:
I guess so, will drop from this PR as the fix should probably be applied to fzf-lua, not plugins that use it

Reply:
Still would need to skip the modules from the `vimPlugin` if we want the plugin to be buildable so we don't block it from that bug. Would just do similar to the `nvim-notify` PR and link to the issue.